packages/lib/package.json

```diff
@@ -24,6 +24,7 @@
 	"dependencies": {
 		"@oclif/errors": "1.3.3",
 		"@smartthings/core-sdk": "^1.3.3",
+		"@types/eventsource": "^1.1.5",
```

Comment:
Maybe I'm missing something but I don't see any reason for this to be a runtime dependency.

Reply:
It needs to be included in dependencies so that it is provided for consuming projects. Otherwise, those projects would see compilation errors even if they aren't explicitly using the SSE stuff.

Reply:
I thought of that, but since this affects projects even if they do not use/care about the SSE related functionality, I didn't want to push that responsibility down to consumers.